File: CyberHunters/Assets/_SPECTRUM/scripts/On.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class On : MonoBehaviour
{
    public GameObject Activate;
    public GameObject button;

    void Update()
    {
        foreach (Transform child in transform)     
        {  
            if (child.gameObject.activeInHierarchy) 
            { 
                if(Activate && button != null)
                {
                    Activate.SetActive(true);
                    button.SetActive(true);
                }
                
            }
        }

       
            
            
         
    }
}

File: Java/Assets/Scripts/On.java
import unityengine.*;

public class On extends MonoBehaviour
{
	public GameObject Activate;
	public GameObject button;


	private void Update()
	{
		for (Transform child : transform)
		{
			if (!child.gameObject.activeInHierarchy)
			{
				return;


			}
			else
			{
				Activate.SetActive(true);
				button.SetActive(true);
			}
		}

	}

}